modules/quant_api/quant_api.module

```diff
@@ -156,6 +156,7 @@ function quant_api_quant_seed($location, $data, $meta = [], $context = []) {
     $on_disk = parse_url($file, PHP_URL_PATH);
     $domain = parse_url($file, PHP_URL_HOST);
 
+    // @fixme Should this be quant_get_base_url() instead of $base_url?
```

Comment:
Yeah this should probably be `quant_get_base_url()` — but the local vs. remote identification should be a little more robust too. We should probably pull that out into a separate issue to refactor and optimise.

Reply:
okay... I can change it for now and make an issue to revisit local vs remote